### DISCUSSION TITLE
toetsing-als-leermoment

### DISCUSSION THREAD

**Discussion by YoussraElmortai** [fdnd/docs.fdnd.nl#84] [Reflectie]
Wij vinden dat er binnen de assements en één op één momenten er aandacht wordt geven aan onze persoonlijke groei en krijgen opbouwende feedback gebaseerd op onze persoonlijke leerdoelen. Ook is er binnen de studie een fijne leeromgeving gecreëerd waar je als student ook van andere studenten leert.

Wij vinden wel dat er soms bepaalde momenten zijn tijdens leertaken waarin wij naar medestudenten worden gestuurd over onze vragen, terwijl het juist handiger is dat die momenten worden opgepakt door leraren. Er zijn natuurlijk altijd wel momenten dat medestudenen dezelfde problemen hebben, maar leraren (vooral tijdens leerjaar 1) zijn even capabel om dit te beantwoorden.

xoxo
Daan, Rick en Youssra

**Comment by ju5tu5** [comment]
Opgenomen in de lopende tekst
